dev-util/kcov/kcov-43.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9,10} )
+
+inherit cmake edo python-any-r1
+
+DESCRIPTION="Kcov is a code coverage tester for compiled languages, Python and Bash"
+HOMEPAGE="https://github.com/SimonKagstrom/kcov/"
+
+if [[ "${PV}" = 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/SimonKagstrom/kcov.git"
+else
+	SRC_URI="https://github.com/SimonKagstrom/kcov/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="GPL-2 MIT"
+SLOT="0"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	dev-libs/elfutils
+	dev-libs/libunistring:=
+	dev-libs/openssl:=
+	net-dns/c-ares:=
+	net-dns/libidn2:=
+	net-libs/libpsl
+	net-misc/curl
+	net-libs/nghttp2:=
+	sys-libs/binutils-libs:=
+	sys-libs/zlib:=
```

Comment:
```suggestion
	sys-libs/zlib
```

Reply:
I'm not sure to understand why I should remove the `:=`. `sys-libs/zlib` has a subslot. Sorry in advance, maybe I missed something. However, I've resolved that.

Reply:
https://github.com/ionenwks/iwdevtools/blob/master/config/qa-vdb.exclude-bind#L5